test/js-api/wasm-module-builder.js

```diff
@@ -217,10 +221,9 @@ let kExprIf = 0x04;
 let kExprElse = 0x05;
 let kExprTry = 0x06;
 let kExprCatch = 0x07;
-let kExprCatchAll = 0x19;
```

Comment:
These old instructions were not deleted; they were just moved below to maintain the sorted order of opcodes.

(This is not actually the same as what v8 file does; it refactored the old opcodes into a list, which I didn't pull in here: https://github.com/v8/v8/blob/8b52e5f026dc859cab827ec584f3eedbc0510459/test/mjsunit/wasm/wasm-module-builder.js#L261-L463. But the sorted order is maintained in this list too.)